.github/workflows/promote_charm.yaml

```diff
@@ -25,5 +37,8 @@ jobs:
         with:
           credentials: ${{ secrets.CHARMHUB_TOKEN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          origin-channel: ${{ github.event.inputs.origin-channel }}
-          destination-channel: ${{ github.event.inputs.destination-channel }}
+          origin-channel: ${{ inputs.origin-channel }}
```

Comment:
Wouldn't it be necessary to add this options here as well? https://github.com/canonical/operator-workflows/blob/main/.github/workflows/publish_charm.yaml#L57

Reply:
Are you referring to the CHARMHUB_TOKEN and GITHUB_TOKEN?
They are set on line 40 and line 41.

Reply:
I think what arturo-seijas is saying is that setting the options added here might also be required for the part he linked to

Reply:
I have looked into that before:
https://github.com/canonical/charming-actions/tree/main/upload-charm
It does not appear to need these new arguments